dts/arm/st/f1/stm32f105.dtsi

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2017 I-SENSE group of ICCS
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <st/f1/stm32f1.dtsi>
```

Comment:
It is, yes, though it doesn't look like the STM32F107xx has Ethernet support in Zephyr at the moment...
- [dts/arm/st/f1/stm32f107.dtsi](https://github.com/zephyrproject-rtos/zephyr/blob/master/dts/arm/st/f1/stm32f107.dtsi)
- [dts/arm/st/f1/stm32f107Xc.dtsi](https://github.com/zephyrproject-rtos/zephyr/blob/master/dts/arm/st/f1/stm32f107Xc.dtsi)

Reply:
Ok.
On stm32 family, we try to take advantage of SoC commonalities to optimize size of code and ease maintenance of dtsi files. 
According to that model, in order to add STM32F105 description, we should have stm32f107.dtsi renamed as stm32f105.dtsi and new stm32f107.dtsi file that includes stm32f105.dtsi. And once ethernet will be supported on stm32f1 series we'll add eth node in stm32f107 node (and until that happen, it will remain empty).